accounts/views.py

```diff
@@ -61,17 +61,17 @@ def user_data(request, username):
         return redirect('home')
 
     usr = get_object_or_404(User, username=username)
-    user_info = get_object_or_404(User_information,  Email=usr.email)
+    user_info = get_object_or_404(UserInformation,  email=usr.email)
 
     ret_all_pics = []
-    all_pictures = Profile_Picture.objects.all()
+    all_pictures = ProfilePicture.objects.all()
     for p in all_pictures:
-        ret_all_pics.append(( p , "/"+ p.Picture.url ))
+        ret_all_pics.append(( p , "/"+ p.picture.url ))
 
 
-    if ( user_info.Picture != None ):
-        pic = Profile_Picture.objects.get(id=user_info.Picture.id)
-        ret_pic = ( "/"+ pic.Picture.url , True, pic.id )
+    if ( user_info.picture != None ):
```

Comment:
Este comentario también se lo hice a @JeanPaulYazbek en otro CR. Nunca hagan este tipo de cosas (yo sé que probablemente ya estaba). Parab eso ponemos una guardia  `if not user_info.picture` (sin los paréntesis). Mucho más claro y conciso.

Reply:
Listo modificado esto, que igualmente era del codigo viejo @mfaria724